storm-core/dependency-reduced-pom.xml

```diff
@@ -0,0 +1,743 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
This file is a build byproduct and should not be checked in.  If you could remove it from this pull request add it to the .gitignore file that would be great.